templates/base-secrets.json

```diff
@@ -60,4 +60,12 @@
     "user": "user - FILL_ME_IN",
     "pass": "pass - FILL_ME_IN"
   }
-}
+},
```

Comment:
i couldn't come up with a better place to put these than the secrets.json file. Hopefully the IP lists don't change too much.